testdata/csharp/csharp_10-dotnet_6/devbox.json

```diff
@@ -0,0 +1,4 @@
+{
+  "packages": [],
+  "shell": {}
```

Comment:
Is `devbox init` adding this by default? (if yes, could you add an `omitempty` to it really quickly?

Reply:
🤔 looks like `omitempty` is already specified in `devbox.Config` for Shell.

I think we may have to make a struct-pointer for that:
https://stackoverflow.com/questions/18088294/how-to-not-marshal-an-empty-struct-into-json-with-go